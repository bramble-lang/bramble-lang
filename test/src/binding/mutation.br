
fn my_main() {
    let mut x:i32 := 1;
    let mut y:i32 := 5;
    printiln x;
    printiln y;
    mut x := 13;
    mut y := y * x;
    printiln x;
    printiln y;
    mut x := -13;
    mut y := -y;
    printiln x;
    printiln y;
    return;
}