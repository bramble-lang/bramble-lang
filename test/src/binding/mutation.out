1
5
13
65
-13
-65
